sdks/python/apache_beam/runners/common.py

```diff
@@ -98,13 +97,9 @@ def __init__(self,
       self.is_new_dofn = True
 
       # SideInputs
```

Comment:
Can you add a docstring for the constructor of DoFnRunner?  In particular, we should note that the expected type for the side_input argument is a sideinputs.SideInputMap.